src/main/java/mate/academy/springboot/datajpa/controller/CategoryController.java

```diff
@@ -0,0 +1,50 @@
+package mate.academy.springboot.datajpa.controller;
+
+import lombok.AllArgsConstructor;
+import mate.academy.springboot.datajpa.dto.mapper.DtoMapper;
+import mate.academy.springboot.datajpa.dto.request.CategoryRequestDto;
+import mate.academy.springboot.datajpa.dto.response.CategoryResponseDto;
+import mate.academy.springboot.datajpa.model.Category;
+import mate.academy.springboot.datajpa.service.CategoryService;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/categories")
+@AllArgsConstructor
+public class CategoryController {
+    private final CategoryService categoryService;
+    private final DtoMapper<CategoryRequestDto, Category, CategoryResponseDto> dtoMapper;
+
+    @PostMapping
+    CategoryResponseDto create(@RequestBody CategoryRequestDto requestDto) {
```

Comment:
If we talk about  RESTful applications, a method which represents by itself an endpoint must be public. Fix it in all places in your code.